x-pack/plugins/secops/server/lib/authentications/query.dsl.ts

```diff
@@ -113,6 +113,13 @@ export const buildQuery = ({
       },
       query: {
         bool: {
+          must: {
+            match_phrase: {
+              'event.action': {
+                query: 'netflow_flow',
```

Comment:
Is this where you wanted this change? Looks like it just breaks authentication table? 

<img width="2173" alt="screen shot 2019-02-11 at 12 48 42 pm" src="https://user-images.githubusercontent.com/1151048/52589171-738a5e80-2dfb-11e9-9df3-56f905e6bc6a.png">

If you wanted this within authentications, wouldn't this be a filter to be more efficient?

Reply:
oups!!! not at all